Input gestures dialog fails to start

<!-- Please read the text in this edit field before filling it in.
Please thoroughly read NVDA's wiki article on how to fill in this template, including how to provide the required files.
Issues may be closed if the required information is not present.
https://github.com/nvaccess/nvda/blob/master/devDocs/githubIssueTemplateExplanationAndExamples.md
Please also note that the NVDA project has a Citizen and Contributor Code of Conduct which can be found at https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md. NV Access expects that all contributors and other community members read and abide by the rules set out in this document while participating or contributing to this project. This includes creating or commenting on issues and pull requests. 

Each of the questions and sections below start with multiple hash symbols (#). Place your answers and information on the blank line below each question.
-->

### Steps to reproduce:
1. try to open imput gestures dialog from settings submenu.

### Actual behavior:
<!-- error persists
ERROR - unhandled exception (10:05:32.409) - MainThread (14120):
Traceback (most recent call last):
  File "gui\blockAction.pyc", line 65, in funcWrapper
  File "gui\__init__.pyc", line 275, in onInputGesturesCommand
  File "gui\blockAction.pyc", line 65, in funcWrapper
  File "gui\__init__.pyc", line 165, in _popupSettingsDialog
  File "gui\inputGestures.pyc", line 589, in __init__
  File "gui\settingsDialogs.pyc", line 201, in __init__
  File "gui\inputGestures.pyc", line 603, in makeSettings
  File "gui\inputGestures.pyc", line 279, in __init__
  File "gui\inputGestures.pyc", line 283, in reset
  File "gui\inputGestures.pyc", line 310, in _fillAllGestures
  File "gui\inputGestures.pyc", line 45, in _getAllGestureScriptInfo
  File "inputCore.pyc", line 671, in getAllGestureMappings
  File "inputCore.pyc", line 691, in __init__
AttributeError: 'NoneType' object has no attribute 'gestureMap'

Use "Speak command keys" (NVDA+4) and speech viewer to copy and paste here. 
Use braille viewer to copy and paste here.
You may additionally include an explanation.
-->
ERROR - unhandled exception (11:42:05.720) - MainThread (14732):
Traceback (most recent call last):
  File "gui\blockAction.pyc", line 65, in funcWrapper
  File "gui\__init__.pyc", line 275, in onInputGesturesCommand
  File "gui\blockAction.pyc", line 65, in funcWrapper
  File "gui\__init__.pyc", line 165, in _popupSettingsDialog
  File "gui\inputGestures.pyc", line 589, in __init__
  File "gui\settingsDialogs.pyc", line 201, in __init__
  File "gui\inputGestures.pyc", line 603, in makeSettings
  File "gui\inputGestures.pyc", line 279, in __init__
  File "gui\inputGestures.pyc", line 283, in reset
  File "gui\inputGestures.pyc", line 310, in _fillAllGestures
  File "gui\inputGestures.pyc", line 45, in _getAllGestureScriptInfo
  File "inputCore.pyc", line 671, in getAllGestureMappings
  File "inputCore.pyc", line 691, in __init__
AttributeError: 'NoneType' object has no attribute 'gestureMap'
debug:
Speaking ['Zdarzenia wejścia...', 'd', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (11:44:55.761) - winInputHook (17468):
Input: kb(desktop):downArrow
IO - speech.speech.speak (11:44:55.774) - MainThread (12212):
Speaking ['Ustawienia...', 'u', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (11:44:56.513) - winInputHook (17468):
Input: kb(desktop):upArrow
IO - speech.speech.speak (11:44:56.527) - MainThread (12212):
Speaking ['Zdarzenia wejścia...', 'd', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (11:44:57.530) - winInputHook (17468):
Input: kb(desktop):enter
DEBUG - gui.settingsDialogs.__new__ (11:44:57.541) - MainThread (12212):
Creating new settings dialog (multiInstanceAllowed:False). State of _instances {}
DEBUG - gui.contextHelp.bindHelpEvent (11:44:57.546) - MainThread (12212):
Did context help binding for InputGesturesDialog
ERROR - unhandled exception (11:44:57.553) - MainThread (12212):
Traceback (most recent call last):
  File "gui\blockAction.pyc", line 65, in funcWrapper
  File "gui\__init__.pyc", line 275, in onInputGesturesCommand
  File "gui\blockAction.pyc", line 65, in funcWrapper
  File "gui\__init__.pyc", line 165, in _popupSettingsDialog
  File "gui\inputGestures.pyc", line 589, in __init__
  File "gui\settingsDialogs.pyc", line 201, in __init__
  File "gui\inputGestures.pyc", line 603, in makeSettings
  File "gui\inputGestures.pyc", line 279, in __init__
  File "gui\inputGestures.pyc", line 283, in reset
  File "gui\inputGestures.pyc", line 310, in _fillAllGestures
  File "gui\inputGestures.pyc", line 45, in _getAllGestureScriptInfo
  File "inputCore.pyc", line 671, in getAllGestureMappings
  File "inputCore.pyc", line 691, in __init__
AttributeError: 'NoneType' object has no attribute 'gestureMap'
IO - speech.speech.speak (11:44:57.933) - MainThread (12212):
Speaking ['Input gestures dialog fails to start · Issue #14750 · nvaccess/nvda — Osobisty — Microsoft\u200b Edge', 'okno', CancellableSpeech (still valid)]
IO - speech.speech.speak (11:44:58.110) - MainThread (12212):
Speaking ['Input gestures dialog fails to start · Issue #14750 · nvaccess/nvda — Osobisty — Microsoft\u200b Edge', 'okno', CancellableSpeech (still valid)]
IO - speech.speech.speak (11:44:58.120) - MainThread (12212):
Speaking ['Input gestures dialog fails to start · Issue #14750 · nvaccess/nvda', 'dokument', CancellableSpeech (still valid)]
IO - speech.speech.speak (11:44:58.129) - MainThread (12212):
Speaking ['główne punkt orientacyjny', CancellableSpeech (still valid)]
IO - speech.speech.speak (11:44:58.201) - MainThread (12212):
Speaking ['Comment body', 'pole edycji', 'Leave a comment', CancellableSpeech (still valid), 'pusta']
IO - speech.speech.speak (11:44:58.352) - MainThread (12212):
Speaking ['Comment body', 'pole edycji', 'Leave a comment', CancellableSpeech (still valid), 'pusta']
IO - inputCore.InputManager.executeGesture (11:44:58.378) - winInputHook (17468):
Input: kb(desktop):NVDA+n
DEBUG - gui.SysTrayIcon.evaluateUpdatePendingUpdateMenuItemCommand (11:44:58.380) - MainThread (12212):
Error while removing pending update menu item
Traceback (most recent call last):
  File "gui\__init__.pyc", line 528, in evaluateUpdatePendingUpdateMenuItemCommand
wx._core.wxAssertionError: C++ assertion ""node"" failed at ..\..\src\common\menucmn.cpp(425) in wxMenuBase::Remove(): removing item not in the menu?
IO - speech.speech.speak (11:44:58.501) - MainThread (12212):
Speaking ['NVDA', 'menu', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (11:44:58.761) - winInputHook (17468):
Input: kb(desktop):downArrow
IO - speech.speech.speak (11:44:58.780) - MainThread (12212):
Speaking ['Opcje', 'rozwijane', 'o', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (11:44:58.954) - winInputHook (17468):
Input: kb(desktop):downArrow
IO - speech.speech.speak (11:44:58.967) - MainThread (12212):
Speaking ['Narzędzia', 'rozwijane', 'n', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (11:44:59.218) - winInputHook (17468):
Input: kb(desktop):rightArrow
IO - speech.speech.speak (11:44:59.246) - MainThread (12212):
Speaking ['Podgląd logu', 'l', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (11:44:59.962) - winInputHook (17468):
Input: kb(desktop):enter
DEBUG - gui.contextHelp.bindHelpEvent (11:44:59.974) - MainThread (12212):
Did context help binding for LogViewer

### Expected behavior:
<!--
Use "Speak command keys" (NVDA+4) and speech viewer to copy and paste here. 
Use braille viewer to copy and paste here.
You may additionally include an explanation.
-->
no error like this persists and gesture imput should open
### NVDA logs, crash dumps and other attachments:

### System configuration
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
version alpha-27914,5182cd98

#### Windows version:
windows 11. 22h2
#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
n/a
### Other questions
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
##yes## If NVDA add-ons are disabled, is your problem still occurring?

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
n/a

--- a/source/braille.py
+++ b/source/braille.py
@@ -2631,7 +2631,8 @@ RENAMED_DRIVERS = {
 	"hid": "hidBrailleStandard",
 }
 
-handler: BrailleHandler
+handler: Optional[BrailleHandler] = None
+
 
 def initialize():
 	global handler

--- a/source/inputCore.py
+++ b/source/inputCore.py
@@ -691,7 +691,7 @@ class _AllGestureMappingsRetriever(object):
 		self.addGlobalMap(manager.userGestureMap)
 		self.addGlobalMap(manager.localeGestureMap)
 		import braille
-		gmap = braille.handler.display.gestureMap
+		gmap = braille.handler.display.gestureMap if braille.handler and braille.handler.display else None
 		if gmap:
 			self.addGlobalMap(gmap)
 

